test/e2e/specs/customer_admin.go

```diff
@@ -81,6 +81,57 @@ var _ = Describe("Openshift on Azure customer-admin e2e tests [CustomerAdmin][Fa
 		Expect(err).ToNot(HaveOccurred())
 	})
 
+	It("should be able to edit project-request", func() {
+		projectTemplate, err := sanity.Checker.Client.CustomerAdmin.TemplateV1.Templates("openshift").Get("project-request", metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		// modify project template object
+		projectTemplate.SetAnnotations(map[string]string{
+			"azure.openshift.io/reconcile-protect": "true",
+		})
+		_, err = sanity.Checker.Client.CustomerAdmin.TemplateV1.Templates("openshift").Update(projectTemplate)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	// Flow:
+	// 1. Read rolebinding with default system:authenticated:oauth value
+	// 2. Update it with 2 values (group + system:authenticated:oauth)
+	// 3. Commit to API
+	// 4. Confirm what we committed is still in api/etcd
+	It("should be able to edit self-provisioners clusterrolebinding", func() {
+		crbOriginal, err := sanity.Checker.Client.CustomerAdmin.RbacV1.ClusterRoleBindings().Get("self-provisioners", metav1.GetOptions{})
```

Comment:
I am trying to understand if there is a scenario in which customer admin can lock themselves out.  If customer admin removes everyone from self-provisioner, is it able to add anyone back in?